tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -191,18 +191,20 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
   dataSeries$?: Observable<ScalarCardDataSeries[]>;
   chartMetadataMap$?: Observable<ScalarCardSeriesMetadataMap>;
   linkedTimeSelection$?: Observable<TimeSelectionView | null>;
-  minMaxSteps$?: Observable<MinMaxStep>;
   columnHeaders$?: Observable<ColumnHeaders[]>;
 
-  lineChartZoom$ = new BehaviorSubject<MinMaxStep>({
-    minStep: -Infinity,
-    maxStep: Infinity,
-  });
-
   onVisibilityChange({visible}: {visible: boolean}) {
     this.isVisible = visible;
   }
 
+  readonly minMaxSteps$ = new BehaviorSubject<MinMaxStep>({
+    minStep: -Infinity,
+    maxStep: Infinity,
+  });
+  readonly lineChartZoom$ = new BehaviorSubject<MinMaxStep>({
+    minStep: -Infinity,
+    maxStep: Infinity,
+  });
```

Comment:
Can you break this default value out into a constant like I am doing in go/tbpr/5938?

Reply:
I am not sure what you are referring to in that PR. However, I made a constant these values.